app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,9 @@
+class MoviesController < ApplicationController
+  def index
+    if params[:genre].present?
+      @movies = Movie.where(genre: params[:genre]).page(params[:page])
+    else
+      @movies = Movie.where(genre: ["Basic", "Git", "HTML&CSS", "Ruby", "Ruby on Rails"]).page(params[:page])
+    end
+  end
```

Comment:
表示数も指定しておきましょう。元アプリは18ですが，動画数が少ないので12にしておきましょうか。

```suggestion
PER_PAGE = 12

  def index
    if params[:genre].present?
      @movies = Movie.where(genre: params[:genre]).page(params[:page])
    else
      @movies = Movie.where(genre: ["Basic", "Git", "HTML&CSS", "Ruby", "Ruby on Rails"]).page(params[:page]).per(PER_PAGE)
    end
  end
```